common/src/main/java/com/microsoft/identity/common/internal/providers/microsoft/azureactivedirectory/ClientInfo.java

```diff
@@ -70,7 +68,7 @@ public ClientInfo(@NonNull String rawClientInfo) throws ServiceException {
         }
 
         // decode the client info first
-        final String decodedClientInfo = new String(Base64.decode(rawClientInfo, Base64.URL_SAFE), Charset.forName(StringExtensions.ENCODING_UTF8));
+        final String decodedClientInfo = Base64.decode(rawClientInfo);
```

Comment:
This should have been throwing an encoding exception.  We'd never see it because the JVM, by definition, ships with UTF-8 these days, but I'm a little confused by the fact that the method signature didn't change.

Reply:
@AdamBJohnsonx  the lower line is Base64 class that wraps around the Apache library. The upper is Android's. They just share the same names.

Reply:
The thing that could have thrown before was Charset.forName.  I suppose that checked exception moved into the Base64 wrapper?